unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotComponentView.cs

```diff
@@ -96,6 +100,8 @@ public override void OnFocus()
         {
             focusedImage.enabled = true;
             tooltipContainer.SetActive(true);
+            focusedImage.transform.localScale = new Vector3(0, 0, 0);
+            focusedImage.transform.DOScale(1, ANIMATION_TIME).SetEase(Ease.OutBack);
```

Comment:
Can we extract all the animations logic into separated methods? In this way we make the code cleaner and understandable. Wdyt?